packages/react/src/Image/Image.tsx

```diff
@@ -0,0 +1,82 @@
+import React from 'react'
+import clsx from 'clsx'
+import styles from './Image.module.css'
+
+type AspectRatio = '1:1' | '16:9' | '16:10' | '4:3' | 'custom'
+
+export type ImageProps = React.ImgHTMLAttributes<HTMLImageElement> & {
+  src: string
+  alt: string
+  ref?: React.Ref<HTMLImageElement>
+  as?: 'picture' | 'img'
+  aspectRatio?: AspectRatio
+  media?: string
```

Comment:
How would we handle multiple `media` declarations in usage? I think this works for one source but not more?

Take this example from w3:

```html
<picture>
  <source media="(min-width:650px)" srcset="img_pink_flowers.jpg">
  <source media="(min-width:465px)" srcset="img_white_flower.jpg">
  <img src="img_orange_flowers.jpg" alt="Flowers" style="width:auto;">
</picture>
```

Also this work for instances where you set `media` but not `srcSet` against it? I think they need to be more tightly coupled. Wonder if this could be a `sources` array, which takes in objects, which are later transformed into individual`<source>` elements internally. 

Alternatively, make children more composable and read them from the `children`.

cc. @danielguillan @josepmartins for additional perspectives.

Reply:
I like the sources array here. I think adding children unnecessarily would overcomplicate the component. Would still love to hear what others think.

Reply:
I wonder if we should make it strict to either apply the sources array or the srcSet and media attributes. What do you all think? If we allow both someone could overset it. Although I tested it and it doesn't break anything.

Reply:
We have agreed to use sources as the property to set when creating these source elements. These will only be possible when picture is the as type. I have updated the component to reflect this.